source/learntris.py

```diff
@@ -27,7 +27,11 @@
                            'T': (lambda: myGame.set_active_tetramino('T')),
                            ')': (lambda: myGame.active_tetramino.rotate_cw()),
                            ';': (lambda: print('\n', end='')),
-                           'P': (lambda: myGame.print_spawn_matrix())}
+                           # debug stuff
+                           # 'P': (lambda: print(type(myGame.active_tetramino.tetramino_matrix), 
```

Comment:
This statement showed that myGame.PLAY_AREA[0] is of type `list` here too. But something happens to it inside method `print_spawned_tetramino()`